### PR TITLE
EET-216 Retire ZW sample dev token

### PR DESCRIPTION
**Pull request by ihorserba**

### CODE REVIEW

File: AUTH.md [hunk 1]
```diff
@@ -2,9 +2,7 @@
 
 To use the API or the SDK you'll need a valid access token. This document describes how to get one.
```
[the remark this answers is not in the document]
Comment (ihorserba) [reply AUTH.md]: changed

File: AUTH.md [hunk 2]
```diff
@@ -2,9 +2,7 @@
...
 
-## Generate API keys and development token
-
-### For Zello consumer network
+## Generate API keys and development token for Zello consumer network
```
[the remark this answers is not in the document]
Comment (ihorserba) [reply AUTH.md]: changed

File: AUTH.md [hunk 3]
```diff
@@ -13,14 +11,6 @@ To use the API or the SDK you'll need a valid access token. This document descri
...
-
-1. Sign into your ZelloWork network at https://`network name`.zellowork.com/
-2. Navigate to https://`network name`.zellowork.com/__keys.html__
-3. Click __Create Key__
-4. Copy and save __Sample Development Token__, __Issuer__, and __Private Key__. Make sure you copy each of the values completely using Select All.
-5. Click __Close__
-
 ## Using sample development token
 
 The developer token you received is valid for 30 days and can be used in your app to connect to API making it easy to test without building your own server for provisioning the actual tokens. Pass the sample development token as `auth_token` when performing logon.
```
[the remark this answers is not in the document]
Comment (ihorserba) [reply AUTH.md]: changed

File: API.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ This document is intended for developers interested in implementation of their o
 
 This API supports subset of Zello features and currently focused on sending and receiving channel voice messages. See [Supported features](#supported-features) for the complete list.
 
-To access the API you need to generate a valid access token, based on [JWT](https://jwt.io/) standard. See [Authentication](#authentication).
+To access the API you need to generate a valid access token (optional for Zello Work), based on [JWT](https://jwt.io/) standard. See [Authentication](#authentication).
```
Comment (vocoded) [inline API.md:13]:
```suggestion
To access the API you will need valid account credentials and/or a valid access token, based on the [JWT](https://jwt.io/) standard. See [Authentication](#authentication).
```

Comment (vocoded) [reply API.md:13]: please also remove the reference to tokens for Zello Work on line 40 below

Comment (ihorserba) [reply API.md]: Changed

File: AUTH.md [hunk 5]
```diff
@@ -1,29 +1,19 @@
...
 
-To use the API or the SDK you'll need a valid access token. This document describes how to get one.
+To use the API or the SDK with Zello Consumer you'll need a valid access token. This document describes how to get one.
 
 ## Generate API keys and development token
 
-### For Zello consumer network
-
 1. Go to https://developers.zello.com/ and click __Login__
 2. Enter your Zello username and password. If you don't have Zello account [download Zello app](https://zello.com/personal/download/) and create one.
```
Comment (vocoded) [inline AUTH.md:8]:
```suggestion
2. Enter your Zello username and password. If you don't have Zello account [download the Zello app](https://zello.com/personal/download/) and create one.
```

**Pull request merged**